src/components/Layout/Main.tsx

```diff
@@ -13,12 +13,14 @@ import { Page } from './Page'
 export type MainProps = {
   titleComponent?: ReactNode
   headerComponent?: ReactNode
+  hideBreadcrumbs?: boolean
 } & ContainerProps
 
 export const Main: React.FC<MainProps> = ({
   children,
   titleComponent,
   headerComponent,
+  hideBreadcrumbs = false,
```

Comment:
If we're going to give this a default value, might as well make this `showBreadCrumbs`, which is usually better ergonomics to avoid double-negatives mental shaenanigans

Reply:
On the other hand the fact that we can pass the `hideBreadcrumbs` shorthand vs. `showBreadcrumbs` would be a valid counterargument, happy either way